bindata/network/ovn-kubernetes/managed/004-config.yaml

```diff
@@ -133,6 +139,12 @@ data:
 {{- if .OVN_MULTI_NETWORK_ENABLE }}
     enable-multi-network=true
 {{- end }}
+{{- if .OVN_NETWORK_SEGMENTATION_ENABLE }}
+    {{- if not .OVN_MULTI_NETWORK_ENABLE }}
+        enable-multi-network=true
```

Comment:
NIT: this indentation is not correct, this is at the same level as other fields below and above

Reply:
hmmm I don't see it.. I must be blind...what I have is:
```
{{- if .OVN_NETWORK_SEGMENTATION_ENABLE }}
    {{- if not .OVN_MULTI_NETWORK_ENABLE }}
        enable-multi-network=true
    {{- end }}
    enable-network-segmentation=true
{{- end }}
```
enable-multi-network needs to be same level as which line?

Reply:
The indentation of the rendered value is not at the same level as others, this would make it the same:
```
{{- if .OVN_NETWORK_SEGMENTATION_ENABLE }}
    {{- if not .OVN_MULTI_NETWORK_ENABLE }}
    enable-multi-network=true
    {{- end }}
    enable-network-segmentation=true
{{- end }}
```
I know that your approach looks better but in the rendered file we would end up with:
```
    [ovnkubernetesfeature]
    enable-egress-ip=true
    enable-egress-firewall=true
    enable-egress-qos=true
    enable-egress-service=true
    egressip-node-healthcheck-port=666
        enable-multi-network=true
    enable-network-segmentation=true
```

Reply:
ahhhhh I see :) gotcha